### PR TITLE
Metas 26-61

### PR DESCRIPTION
**Pull request by GranSpencer09**

### CODE REVIEW

**Comment by GranSpencer09** [comment]
@michaelferranti

**Comment by GranSpencer09** [comment]
@michaelferranti

File: cloud/aws/asg.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ sidebar: home_sidebar
 redirect_from:
   - /cloud/aws-ec2-asg.html
   - /portworx-on-aws-asg.html
+  meta-description: "Learn to scale a Portworx cluster up or down on AWS with Auto Scaling. Use our tips and tricks to make it simple!"
```
Comment (msh100) [inline cloud/aws/asg.md]: The indentation needs to be removed from here

**Comment by msh100** [comment]
Merged gh-pages into #125

**Pull request closed without merging**